SCClassLibrary/Common/Control/Server.sc

```diff
@@ -357,7 +357,8 @@ Server {
 		addr = netAddr ?? { NetAddr("127.0.0.1", 57110) };
 		inProcess = addr.addr == 0;
 		isLocal = inProcess || { addr.isLocal };
-		remoteControlled = isLocal;
+		remoteControlled = isLocal.not;
+		freeAllFreesRootNode = isLocal;
```

Comment:
maybe better this: 

`freeAllFreesRootNode = freeAllFreesRootNode ? isLocal`

and leave empty above.

Or, alternatively, have no setter.